### PR TITLE
update zh translation /docs/reference/setup-tools/kubeadm/generated/k…

### PR DESCRIPTION
**Pull request by tanjunchen**

### CODE REVIEW

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_token.md [hunk 1]
```diff
@@ -115,11 +132,33 @@ kubeadm token [flags]
...
+查看其它
+
+<!--
+* [kubeadm](kubeadm.md)	 - kubeadm: easily bootstrap a secure Kubernetes cluster
+* [kubeadm token create](kubeadm_token_create.md)	 - Create bootstrap tokens on the server
+* [kubeadm token delete](kubeadm_token_delete.md)	 - Delete bootstrap tokens on the server
+* [kubeadm token generate](kubeadm_token_generate.md)	 - Generate and print a bootstrap token, but do not create it on the server
+* [kubeadm token list](kubeadm_token_list.md)	 - List bootstrap tokens on the server
+-->
+* [kubeadm](kubeadm.md)	 - kubeadm：轻易建立安全 Kubernetes 集群
```
Comment (hwdef) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_token.md]:
```suggestion
* [kubeadm](kubeadm.md)	 - kubeadm：轻易建立一个安全的 Kubernetes 集群
```

**Comment by hwdef** [comment]
/lgtm
Thanks for your translation!

**Comment by xichengliudui** [comment]
/lgtm
approve

**Pull request merged**